c/logging.c

```diff
@@ -685,14 +799,41 @@ void zowelog(LoggingContext *context, uint64 compID, int level, char *formatStri
       lastResortLog(message);
       return;
     }
+
+    ACEE *acee;
+    acee = getAddressSpaceAcee();
+    char user[7] = { 0 }; // wil this always be 7?
+    snprintf(user,7,acee->aceeuser+1);
+
+    char *locationInfo; // largest possible variation in size
+    getLocationData(path,line,&locationInfo,compID,component); // location info is allocated in getLocationData
+
+    char logLevel[8] = { 0 }; // logLevel will be a max of 8 characters
+    getLevel(level, logLevel);
+
+    char timestamp[24] = { 0 }; // formatted date is 24 characters
+    getLogTimestamp(timestamp); // UTC time
+
+    pthread_t threadID = pthread_self();
+    char thread[10];
+    snprintf(thread,10,"%d",threadID);
+
+    //    timestamp + " <ZWESA1:thread> " + user + log level + locationInfo + " " + format string
+    int newSize = 24 + 11 + 10 + 7 + strlen(logLevel) + strlen(locationInfo) + 1 + strlen(formatString); //accounts for formatting I know it's garbage
+    char *newFormatString = (char *) safeMalloc(sizeof(char) * newSize,"newFormatString");
```

Comment:
What is the point of `sizeof(char) * newSize` when sizeof(char) == 1? newSize is sufficient.